org.lflang/src/org/lflang/federated/generator/FedConnectionInstance.java

```diff
@@ -0,0 +1,123 @@
+package org.lflang.federated.generator;
+
+import org.lflang.federated.serialization.SupportedSerializers;
+import org.lflang.generator.PortInstance;
+import org.lflang.generator.RuntimeRange;
+import org.lflang.generator.SendRange;
+import org.lflang.lf.Connection;
+
+/**
+ * Class representing a federated connection.
+ */
+public class FedConnectionInstance {
+
+    /**
+     * FIXME
```

Comment:
Lots of FIXMEs

Reply:
It was left like this by the author. I'll make an attempt at fixing it.